frontend/src/components/Aiboard.vue

```diff
@@ -46,12 +46,14 @@ export default {
         let moves = this.game.moves({ verbose: true });
         let randomMove = moves[Math.floor(Math.random() * moves.length)];
         this.game.move(randomMove);
-        let move = {
-          orig: randomMove.from,
-          dest: randomMove.to,
-          color: randomMove.color
-        };
-        this.updateHistory(move);
+        if (this.game.history().length % 2 === 0) {
+          let move = [
+            this.game
+              .history({ verbose: true })
+              .slice(this.game.history().length - 2, this.game.history().length)
+          ];
+          this.updateHistory(move);
+        }
```

Comment:
Move this code into GameHistory component to avoid duplications. Or create a function and put it in the parent chessboard component.